server/template.go

```diff
@@ -78,3 +82,65 @@ input:focus::-moz-placeholder{color:transparent}
 input:focus:-moz-placeholder{color:transparent}
 input:focus:-ms-input-placeholder{color:transparent}`
 )
+
+// indexJS is the javascript for the main index page.
+func indexJS(defaultProvider string) string {
+	return `function validateForm() {
+    var v = document.forms["search"]["q"].value;
```

Comment:
Where is search defined? I didn't think there was any special handling for forms -> vars.